Server/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -64,7 +65,7 @@ void CLuaPedDefs::LoadFunctions()
         {"warpPedIntoVehicle", WarpPedIntoVehicle},
         {"removePedFromVehicle", RemovePedFromVehicle},
         {"setPedDoingGangDriveby", SetPedDoingGangDriveby},
-        {"setPedAnimation", SetPedAnimation},
+        {"setPedAnimation", ArgumentParser<SetPedAnimation>},
```

Comment:
`ArgumentParserWarn<false, SetPedAnimation>`. Otherwise it will be backward incompatible.

Reply:
What exactly do you mean? I tried several different examples and they all worked

Reply:
`ArgumentParser<>` will generate a Lua error if any exception will be thrown. `ArgumentParserWarn<false, ...>` will return **false** in the same situation. Previously created scripts expect a **false**, not a Lua error. That's why `ArgumentParser<>` makes this function backward incompatible.

Reply:
Yes, in the case of ArgumentParser nil is returned instead of false. Thanks for the clarification